osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -126,11 +126,19 @@ public void PrepareTrackForPreviewLooping()
         }
 
         /// <summary>
-        /// Transfer a valid audio track into this working beatmap. Used as an optimisation to avoid reload / track swap
-        /// across difficulties in the same beatmap set.
+        /// Attempts to transfer the audio track to a target working beatmap, if valid for transferring.
+        /// Used as an optimisation to avoid reload / track swap across difficulties in the same beatmap set.
         /// </summary>
-        /// <param name="track">The track to transfer.</param>
-        public void TransferTrack([NotNull] Track track) => this.track = track ?? throw new ArgumentNullException(nameof(track));
+        /// <param name="target">The target working beatmap to transfer this track to.</param>
+        /// <returns>Whether the track is valid and has been transferred to this working beatmap.</returns>
```

Comment:
```suggestion
        /// <returns>Whether the track has been transferred to the <paramref name="target"/>.</returns>
```

Dunno why but the "validity" part of the original xmldoc is throwing me off here. The latter change is fixing a straight-up error (transfer is happening *away* from this instance, not *to* it).

Reply:
Whoops, missed to reword this part while I was changing direction of this method.